drivers/wifi/nrfwifi/inc/fmac_main.h

```diff
@@ -83,6 +83,7 @@ struct nrf_wifi_vif_ctx_zep {
 #ifdef CONFIG_NRF_WIFI_RPU_RECOVERY
 	struct k_work nrf_wifi_rpu_recovery_work;
 #endif /* CONFIG_NRF_WIFI_RPU_RECOVERY */
+	unsigned int rts_threshold_value;
```

Comment:
Why not define as `int` instead of `unsigned int` (see comment below in `wifi_mgmt.c`) ?